### ISSUE TITLE
undefined method `[]' for nil:NilClass

### ISSUE THREAD

**Issue by ghost** [negativecode/vines#25]
After installing everything and running vines-services init app.com

I get "undefined method `[]' for nil:NilClass" after putting in the name and password of my first user. 

Any ideas?

**Comment by Strech** [comment]
@chemicalfire Plz show a full backtrace

**Comment by ghost** [comment]
Pardon my ignorance but can you explain how I would do that. I was trying to figure out how to do that, I tried -l output.log but wasn't getting anything.

**Comment by Strech** [comment]
@chemicalfire Start not demonized vines, like `vines start`, repeat this error again and just copy and paste all trace from console. That's all :joy:

**Comment by ghost** [comment]
Well I get the error when initializing the application using 

"vines-services init app.com" 

So its even before I hope into the folder and start vines. 

I managed to get it working on a second VM, but I still have the original VM I can get the output from if I can figure out how to get the stack trace from the init command. Or I can just close this if you don't think its an issue with vines.

**Comment by ghost** [comment]
Well actually I was able to reproduce it. 

So after installing the gems, I ran vines-services init app.com

It worked, but then I deleted that folder and tried running the init command a second time and got the error after entering my username and password for the initializing account.

**Comment by Strech** [comment]
@chemicalfire Could you try execute something like

``` bash
$ gem install vines
$ vines init app.com
```

**Comment by Strech** [comment]
@chemicalfire I'm not a vines dev, and i init my vines app via `vines` binary, not `vines-services` ... may be you missed something in vines-services config file, etc

**Comment by ghost** [comment]
Yeah I was able to do vines init app.com just fine, that is my plan for continuing. But I figured since it crashed without an error on the vine-services init I should report it. 

Thanks for providing help, It is appreciated.

**Comment by Strech** [comment]
@chemicalfire Thanks :star2:

Maybe i take couple minutes by looking at vines-services code, and if i found something, i will write to you

**Comment by arlyxiao** [comment]
I have the same problem. it's what I do below:
1, Run vines-services init wonderland.lit  to create an account 'alice' and the dir wonderland.lit
2, remove the dir wonderland.lit
3 Run  vines-services init wonderland.li again and type alice to create the user. then the error happens
if I type other username which never used before, it will have no the error

**Comment by nordringrayhide** [comment]
+1 having the same issue and workflow that @arlyxiao have described

**Comment by nordringrayhide** [comment]
So looks like I've found the drawback. if you want fully remove initialised directory, you have to do the same with its database as well. 

```
rm -rf /usr/local/var/lib/couchdb/wonderland_lit.couch
```

but be careful you've stop couchdb server before removing the database or just restart db server before initialisation of the dir wonderland.lit

Naturally, database location depends on your install CouchDB use case.
